### ISSUE TITLE
Kernen en regionale woningmarkten

### ISSUE THREAD

**Issue by Tine-VB** [provinciesincijfers/PinC-taakopvolging#400]
Valerie:
Bij ons willen de collega's ruimtelijke planning i.k.v. BRL graag indicatoren opvragen op niveau van kernen en regionale woningmarkten (dit zijn telkens aggregaties van statistische sectoren). Bv. evolutie inwoners binnen de kernen en buiten het gebied van de kernen. Het gaat om 10 regionale woningmarkten, 43 bovenlokale kernen en 239 lokale kernen in Limburg. Momenteel zijn deze gebiedsniveaus opgeladen in LinC, maar zou handiger zijn om de gegevens vanuit PinC te kunnen opvragen.

**Comment by Tine-VB** [comment]
Ik vroeg even na bij onze dienst ruimtelijke planning, daar is ook veel interesse: We zijn heel erg bezig met het kernversterkingsbeleid concreet vorm te geven – onder andere RUPs die de kernen juridisch afbakenen om zo het beleid écht te kunnen differentiëren. Daar dan ook eenvoudig gegevens en analyses aan kunnen koppelen is zeer nuttig.
Ik meen dat de statistische sectoren meestal relatief goede benaderingen zijn van de kernen – lichte afwijkingen door recente verkavelingen daargelaten, dus dat is zeker bruikbaar. Maar nog beter zou zijn om op termijn het statistische en beleidsmatige op elkaar af te stemmen door echt de juridische afbakeningen te gebruiken waar ze al bestaan. Op dit moment zijn er nog niet zoveel definitief (en dan nog eens redelijk verspreid), maar dit zou de komende jaren (liefst sterk) moeten toenemen.

**Comment by Tine-VB** [comment]
Is er eigenlijk al een werkgroep 'Ruimte' samengesteld/samengekomen?

**Comment by valeriepirard** [comment]
Nee, deze is uitgesteld tot het najaar, maar ik heb wel een lijst van mensen die geïnteresseerd zijn.

**Comment by Tine-VB** [comment]
Dan kan daar eens nagegaan worden of dit beschikbaar is (gebiedsdekkend?) voor alle provincies?

**Comment by HildeC** [comment]
bij ons bestaat er al ruim 10 jaar een kernenkaart (combinatie van kernafbakening en afbakening stedelijk gebied). De kernen en stedelijk gebied volgen niet de contouren van de stat sectoren, dus voor dergelijke oefening gebruiken we eigen kubussen. 
pdf geeft de methodiek die gehanteerd werd om de kernen af te bakenen (combinatie van juridische contour en morfologische contour). bedoeleing was om dat om de x jaar te actualiseren. voor de kernen op zicht is er geen verder tijdspad, voor de provinciaal stedelijke gebeiden lopen weer afbakeningsoefeningen.
[Actualisatie NIS-kernen - Methodiek (2013).pdf](https://github.com/provinciesincijfers/PinC-taakopvolging/files/11870943/Actualisatie.NIS-kernen.-.Methodiek.2013.pdf)

De regionale woonmarkten zijn een samentelling van gemeenten/deelgemeenten. Deze zouden in principe ook in PinC kunnen
